### PR TITLE
[VI] Add `Community/How you can help`

### PR DESCRIPTION
**Pull request by summyvn**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by summyvn** [comment]
If the translation is wrong, I will correct it when I wake up tomorrow morning

File: wiki/Community/How_you_can_help!/vi.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+- Tham gia [dự án beatmap](https://osu.ppy.sh/community/forums/53) để tìm hiểu việc hợp tác với người khác thú vị như thế nào!
+- Bắt đầu dự án của riêng bạn hoặc tham gia dự án của người khác! Cho dù đó là về những chú pippi nhảy múa hay đơn giản là làm thứ gì đó thú vị cho osu!, mọi thứ có thể được chấp nhận!
+ - Kiểm tra [danh sách các dự án phổ biến](/wiki/Community/Projects).
+
+## Đóng góp cho tương lai của osu!
+
+Bắt đầu vào năm 2017, dự án [osu!(lazer)](/wiki/Client/Release_stream/Lazer) là trọng tâm hiện tại của [osu!team](/wiki/People/osu!_team) và cùng với wedsite mới, đó là những gì bạn thấy ngày hôm nay! Nếu bạn muốn làm cho những dự án này trở nên tốt hơn nữa và đến gần hơn với tương lai của osu!, thì bạn có thể giúp một số việc dưới nhưng đây chỉ là một vài ví dụ:
+
+- Giúp đỡ dự án [osu!(lazer)](https://github.com/ppy/osu). Có rất nhiều thứ cần triển khai trên phiên bản mới của osu!, và bạn có thể giúp biến điều đó thành hiện thực!
+- Làm cho [wedsite](https://github.com/ppy/osu-web) ngày càng tốt hơn.
```
Comment (avariation) [inline wiki/Community/How_you_can_help!/vi.md:76]:
```suggestion
- Làm cho [trang web](https://github.com/ppy/osu-web) ngày càng tốt hơn.
```

File: wiki/Community/How_you_can_help!/vi.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+## Tránh tiêu cực! Làm cho mọi người vui vẻ!
+
+Có một câu nói cổ rằng :*“bạn nên trở thành một phần của giải pháp chứ không phải của vấn đề”*. Chỉ cần làm theo câu nói cổ này, bạn sẽ giúp được osu! bằng cách làm cho cộng đồng trở thành một nơi tươi sáng hơn. Mọi người sẽ luôn phàn nàn, nhưng vẫn có rất nhiều cách bạn có thể giúp cải thiện tâm trạng trong cộng đồng:
```
Comment (avariation) [inline wiki/Community/How_you_can_help!/vi.md:80]:
```suggestion
Người xưa có câu "hãy là một phần của giải pháp, đừng là một phần của vấn đề". Chỉ cần làm theo câu nói này, bạn sẽ giúp được osu! bằng cách làm cho cộng đồng trở thành một nơi tươi sáng hơn. Mọi người sẽ luôn phàn nàn, nhưng vẫn có rất nhiều việc bạn có thể làm để giúp cải thiện tâm trạng của mọi người:
```

File: wiki/Community/How_you_can_help!/vi.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+- Hãy thân thiện với nhau; cuộc sống quá ngắn để tranh luận.
+- [Báo cáo những kẻ gian lận khi bạn nhìn thấy chúng](/wiki/Reporting_bad_behaviour/Handling_foul_play). Vui lòng không phàn nàn về chúng thông qua các phương tiện khác vì có một cách đơn giản là`!report <username> seems to be cheating`có thể giúp chúng bay màu trong vòng vài phút đến vài giờ.
```
Comment (avariation) [inline wiki/Community/How_you_can_help!/vi.md:83]:
```suggestion
- [Báo cáo những kẻ gian lận khi bạn nhìn thấy chúng](/wiki/Reporting_bad_behaviour/Handling_foul_play). Vui lòng không phàn nàn về những người này thông qua các phương tiện khác vì chỉ cần một câu đơn giản là `!report <username> seems to be cheating` là đủ để vấn đề của bạn đi khuất mắt chỉ trong vài phút đến vài giờ.
```

File: wiki/Community/How_you_can_help!/vi.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+- Hãy cố gắng hòa giải các tình huống tiêu cực và biến chúng thành kết quả tích cực.
+
+**Mặc dù bài viết này không thể đề cập đến việc bạn có thể giúp được bao nhiêu trong cộng đồng osu!, chỉ cần chơi game cũng có thể giúp osu! thậm chí còn tuyệt vời hơn nữa!**( đừng nghiện game quá, lol)
```
Comment (avariation) [inline wiki/Community/How_you_can_help!/vi.md:86]:
```suggestion
**Mặc dù bài viết này không thể đề cập tất cả các cách để bạn có thể đóng góp cho cộng đồng của osu!, nhưng chỉ cần chơi game thôi cũng có thể giúp osu! thậm chí còn tuyệt vời hơn nữa!**
```

**Review by avariation — CHANGES_REQUESTED** [review]
Review from native speaker.

File: wiki/Community/How_you_can_help!/vi.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+Nếu bạn thấy ai đó đang gặp vấn đề với game, hãy giúp họ một tay! Cho dù đó là vấn đề về phần cứng, câu hỏi về game hoặc beatmap, nếu bạn có thể trả lời được thì bạn có thể giúp họ giải quyết vấn đề.
+
+Sau đây là các gợi ý để bạn có thể bắt đầu giúp đỡ người khác trên diễn đàn:
+
+- Chọn một diễn đàn thu hút sự quan tâm của bạn, ví dụ:
+ - [Help (Diễn đàn Trợ giúp)](https://osu.ppy.sh/community/forums/5)
+ - [Mapping Discussion (Diễn đàn Thảo luận Beatmap)](https://osu.ppy.sh/community/forums/56)
+ - [Storyboarding](https://osu.ppy.sh/community/forums/20)
+ - [skinning](https://osu.ppy.sh/community/forums/15)
```
Comment (avariation) [inline wiki/Community/How_you_can_help!/vi.md:33]:
```suggestion
 - [Skinning](https://osu.ppy.sh/community/forums/15)
```

File: wiki/Community/How_you_can_help!/vi.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+ - [Gameplay & Rankings](https://osu.ppy.sh/community/forums/13)
+- Đọc qua các bài đăng được ghim ở diễn đàn để tìm hiểu các quy tắc cụ thể dành riêng cho từng diễn đàn nhỏ và những thông tin hữu ích ở trong bài đăng đó.
+-Sử dụng nút `Mark forum as read` ở trên cùng.
```
Comment (avariation) [inline wiki/Community/How_you_can_help!/vi.md:36]:
```suggestion
- Sử dụng nút `Mark forum as read` ở trên cùng.
```

File: wiki/Community/How_you_can_help!/vi.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+- Theo dõi tất cả các chủ đề mới, đọc từng chủ đề và trợ giúp khi có thể  — bạn có thể sử dụng biểu tượng chưa đọc để chắc chắn rằng bạn bắt kịp với mọi thông tin.
```
Comment (avariation) [inline wiki/Community/How_you_can_help!/vi.md:37]:
```suggestion
- Theo dõi tất cả các chủ đề mới, đọc từng chủ đề và trợ giúp khi có thể — bạn có thể sử dụng biểu tượng chưa đọc để chắc chắn rằng bạn bắt kịp với mọi thông tin.
```

**Review by avariation — CHANGES_REQUESTED** [review]
Minor changes

**Review by avariation — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**